### PR TITLE
new download page

### PR DESCRIPTION
**Pull request by nalinbhardwaj**
new flow:
- `/download` instead of `/l/download`: no deep linking into the app
- on non iOS / android devices, redirect to `/download/other` -- show all three possibilities

Similar to signal.org.

TODO:
- [x] Pick an icon to represent both categories that's not bad, cc @michelle-chiu want to send me one and we can merge this

demo on various screen size: https://streamable.com/ip7vgj

### CODE REVIEW

**Comment by michelle-chiu** [comment]
@nalinbhardwaj Looks great :)

The only ask is if you could change the iPhone/iPad icon to this one, so the stroke weight matches the one on desktop? 

smartphone

**Comment by nalinbhardwaj** [comment]
Changed icon to be from pack after discussing with @michelle-chiu IRL

File: apps/daimo-web/src/components/buttons.tsx [hunk 1]
```diff
@@ -29,17 +29,57 @@ export function PrimaryOpenInAppButton({
...
+    case "danger":
+      return "text-danger border-danger";
+    case "success":
+      return "text-success border-success";
+    default:
+      return "text-primaryLight border-primaryLight";
+  }
+}
+
+export function PrimaryButton({
```
Comment (dcposch) [inline apps/daimo-web/src/components/buttons.tsx:62]: 👍

**Review by dcposch — APPROVED** [review]
LGTM

**Pull request merged**